std_msgs/String move_group
geometry_msgs/PoseStamped initial_shelf_pose
---
std_msgs/Bool success
